src/dialogsettings.ui

```diff
@@ -935,6 +935,16 @@
               </property>
              </spacer>
             </item>
+            <item>
```

Comment:
Allowing to change this option via settings UI is not needed. People who can access log files are able to start the app with the appropriate command-line.